config/config.pubsub.extended.hocon

```diff
@@ -111,6 +111,16 @@
   # no assets will be updated if the key is absent
   "assetsUpdatePeriod": "7 days"
 
+  # Optional, configuration of remote adapters
+  # Increasing maxConnections is helpful if your adapter has high latency
+  # remoteAdapters.configs is a list of adapter configurations
+  "remoteAdapters": {
+    "connectionTimeout": "10 seconds",
+    "readTimeout": "45 seconds",
+    "maxConnections": 10,
```

Comment:
Neat: put these in `application.conf` so that they are all optional and it's possible to specify only `configs`